src/main/java/com/griddynamics/cloud/learning/web/dto/SortColumn.java

```diff
@@ -0,0 +1,6 @@
+package com.griddynamics.cloud.learning.web.dto;
```

Comment:
Let's get rid from it, and be flexible enough to process sorting any kind of article fields